### PR TITLE
feat: alert box on new task added successfully

### PR DESCRIPTION
**Pull request by nikhilgarg26**
Alert Box added new feature 
The also need not to reload on each render only the specific elements are rendered.

Fixes: #35

### CODE REVIEW

**Review by malivinayak — CHANGES_REQUESTED** [review]
image
https://github.com/malivinayak/To-Do-Manager/issues/35#issue-1920826381

image

**Comment by nikhilgarg26** [comment]
i have changed the alert icon for the dialog box

**Comment by malivinayak** [comment]
> image

instead of warning sign you can use tick mark as a sign of success

**Comment by nikhilgarg26** [comment]
changes to color have been made

File: index.html [hunk 1]
```diff
@@ -32,6 +33,15 @@ <h1>Task List ↴ </h1>
           </form>
           </header>
     </div>
+
+
+    <div class="alertt hide" id="custAlert3">
+      <span class="msgt">Task added Successfully</span>
```
Comment (malivinayak) [inline index.html:39]: Add following code in between
```html
<span class="fas fa fa-check-circle" style="font-size:24px"></span>
```      

| Prev: | After update: |
|---|---|
| image | image |

**Review by malivinayak — CHANGES_REQUESTED** [review]
You have make a good color combination.
I have attached a code to add tick mark for alert message.
Add that code and I will merge the PR.

Thank You

File: index.html (hunk 1 above)
Comment (malivinayak) [reply index.html:39]: In after update screenshot tock mark is added which looks more appropriate.

**Comment by malivinayak** [comment]
> changes to color have been made

Looks great

File: index.html (hunk 1 above)
Comment (nikhilgarg26) [reply index.html:39]: image

**Review by malivinayak — APPROVED** [review]
Nicely done!

**Pull request merged**